fast64_internal/sm64/sm64_geolayout_bone.py

```diff
@@ -228,27 +237,38 @@ def draw(self, context):
 
         prop_split(col, obj, "geo_cmd_static", "Geolayout Command")
         drawLayerWarningBox(col, obj, "draw_layer_static")
-        col.prop(obj, "use_render_area")
+
+        renderAndShadowBox = col.box()
+        if obj.use_render_range or obj.add_shadow or obj.use_render_area:
+            renderAndShadowBox.label(text="These values are in blender units.")
+            renderAndShadowBox.label(text="These propreties are only applied if they are the root object of an object geolayout.")
+
+        if obj.add_shadow or obj.use_render_area:
+            renderAndShadowBox.label(text="For armature geolayouts, see the armature's object properties instead.")
+
+        renderAreaBox = renderAndShadowBox.box()
+        renderAreaBox.prop(obj, "use_render_area")
         if obj.use_render_area:
-            renderAreaBox = col.box()
-            renderAreaBox.label(text="This is in blender units.")
-            renderAreaBox.label(text="This only applies if this is the root object of an object geolayout.")
-            renderAreaBox.label(text="For armature geolayouts, see the armature's object properties instead.")
-            prop_split(col, obj, "culling_radius", "Culling Radius")
-        col.prop(obj, "use_render_range")
+            prop_split(renderAreaBox, obj, "culling_radius", "Culling Radius")
+
+        renderRengeBox = renderAndShadowBox.box()
```

Comment:
`renderRangeBox` is probably what you meant, but ideally all variables should use lowercase_with_underscores, though specifically UI layout options in blender should be referred to as temporary single word variables. See: https://docs.blender.org/api/current/info_best_practice.html for UI layout naming practices.